utils/tx.ts

```diff
@@ -850,18 +861,22 @@ export async function scheduleBootstrap(
   mainCurrency: BN,
   bootstrapCurrency: BN,
   waitingPeriod: number,
-  bootstrapPeriod: number
+  bootstrapPeriod: number,
+  whitelistPeriod?: number
```

Comment:
in typescript, there is something for default parameters, so you can remove lines 869 - 871 by adding this in the function params specification
whitelistPeriod = 1